GSA_Adapter/GSA_Adapter.csproj

```diff
@@ -111,12 +120,49 @@
   <ItemGroup>
     <Compile Include="AdapterActions\Pull.cs" />
     <Compile Include="AdapterActions\Execute.cs" />
+    <Compile Include="Convert\AdapterID.cs" />
+    <Compile Include="Convert\FromGsa\Elements\Bar.cs" />
+    <Compile Include="Convert\FromGsa\Elements\FEMesh.cs" />
+    <Compile Include="Convert\FromGsa\Properties\LinkConstraint.cs" />
+    <Compile Include="Convert\FromGsa\Loads\Loadcase.cs" />
+    <Compile Include="Convert\FromGsa\Loads\LoadCombination.cs" />
+    <Compile Include="Convert\FromGsa\Properties\Material.cs" />
+    <Compile Include="Convert\FromGsa\Elements\Node.cs" />
+    <Compile Include="Convert\FromGsa\Results\Results.cs" />
+    <Compile Include="Convert\FromGsa\Elements\RigidLink.cs" />
+    <Compile Include="Convert\FromGsa\Properties\SectionProperty.cs" />
+    <Compile Include="Convert\NotSupportedWarning.cs" />
```

Comment:
Can't see how this is a Convert method. I'd have it in the Engine, if anything. The body of the method also does not seem to justify a wrap inside a method.

Reply:
hmm, yeah, get your point. Might be a case where we could move it to the engine. Wrapped it into a method to avoid having to change phrasing/spelling in multiple places and make sure it is consistent.